recipes/nextsilicon-cpp-subprocess/all/test_package/CMakeLists.txt

```diff
@@ -1,14 +1,12 @@
-cmake_minimum_required(VERSION 3.1)
+cmake_minimum_required(VERSION 3.15)
 project(test_package CXX)
 
-include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
-conan_basic_setup()
-
 set(CMAKE_CXX_STANDARD 11)
```

Comment:
Adding that check above, we could remove this one. Conan puts automatically this one through the CMakeToolchain generator.